buildscripts/set_tensorflow_bazelrc.sh

```diff
@@ -17,17 +17,28 @@ ARCH=`uname -p`
 
 ## ARCHITECTURE SPECIFIC OPTIMIZATIONS
 ## These are settings and arguments to pass to GCC for
-## optimization settings specific to the target architecture
+## optimization settings specific to the target CPU architecture
 ##
 OPTION_1=''
 OPTION_2=''
```

Comment:
Do you need to add OPTION_3-OPTION_7 here? I guess you wouldn't need to add them under the PPC section as well.

Reply:
For PPC, should we not try -mvsx or -maltivec? Or power8 enables those two implicitly?